torch/_dynamo/symbolic_convert.py

```diff
@@ -2080,8 +2084,12 @@ def create_call_resume_at(self, inst):
                 name, types.FunctionType(new_code, self.f_globals, name)
             )
             cg.extend_output(cg.load_function_name(name, True, stack_len))
+        from . import utils
 
-        cg.extend_output([cg.create_load(k) for k in argnames])
+        for k in argnames:
+            cg.load_import_from(utils.__name__, "numpy_to_tensor"),
+            cg.extend_output([cg.create_load(k)])
+            cg.extend_output(create_call_function(1, False))
```

Comment:
So this is very bad. The problem is `self.stack` is empty here so can't look into what's in the stack to decide whether we should convert ndarray to tensor.

Reply:
Yeah, this doesn't seem like the right thing to do. Are you still opposed to wrapping the graph so the numpy inputs get translated into tensors before calling into the compiler produced callable instead? I'm not sure if I can offer you better.

Reply:
Addressed comments.